frontend/src/metabase/query_builder/components/filters/FilterPopover/FilterPopover.unit.spec.tsx

```diff
@@ -36,82 +37,70 @@ const QUERY = Question.create({
     ["field", PRODUCTS.TITLE, { "source-field": ORDERS.PRODUCT_ID }],
     "asdf",
   ]);
-
 const [RELATIVE_DAY_FILTER, NUMERIC_FILTER, STRING_CONTAINS_FILTER] =
   QUERY.filters();
 
 const dummyFunction = jest.fn();
 
+const setup = ({
```

Comment:
Added this since it seemed like we were just doing the same thing over and over again on the setup side